classes/war/Operation.php

```diff
@@ -46,13 +49,13 @@ class Operation {
 
     const BASE_OPERATION_INTERVAL_PROGRESS_PERCENT = 20; // progress per interval
     const BASE_OPERATION_INTERVAL_SECONDS = 12; // time per interval
-    const BASE_OPERATION_POOL_COST = 150; // chakra/stam cost per interval, 750 total
+    const BASE_OPERATION_POOL_COST_PER_RANK = 50; // chakra/stam cost per interval per rank, 750 total
```

Comment:
Maybe we can go ahead and specify specific values for each rank in an array here, given how regen rates and max pools don't have a defined curve as you go up the ranks I think that's simpler for us to tune than trying to build a specific curve for this